src/telegrafs/components/TelegrafOutputOverlay.tsx

```diff
@@ -74,7 +74,16 @@ class TelegrafOutputOverlay extends PureComponent<Props> {
       .sort((a, b) => {
         const _a = a.name.toLowerCase()
         const _b = b.name.toLowerCase()
-        return _a > _b ? 1 : _a < _b ? -1 : 0
+
+        if (_a > _b) {
+          return 1
+        }
+
+        if (_a < _b) {
+          return -1
+        }
+
+        return 0
```

Comment:
see above comment...and since this is in 2 places, we should have a utils  method to case-insensitive compare strings in one place.